CHANGES/7498.bugfix

```diff
@@ -0,0 +1 @@
+Increased Distributions'd fields release_short and  base_product_short length.
```

Comment:
Since it's a user facing message, would something like "Fixed `value too long` error for distribution tree sync." make sense?